run

```diff
@@ -22,25 +26,58 @@ parser.add_argument(
     action="store_true",
     help="Run container in background & don't show container logs",
 )
+# Set to false if your current user is already in the docker user group
+parser.add_argument(
+    "--sudo",
+    default=False,
+    action="store_true",
+    help="Run commands with sudo",
+)
 args = parser.parse_args()
 
+use_sudo = args.sudo
+if not args.sudo:
+    current_user = getpass.getuser()
+    user_in_docker_group = current_user in grp.getgrnam("docker").gr_mem
+    if user_in_docker_group:
+        print("{} is in docker group".format(current_user))
+    #Run docker ps without sudo to check if it can be used without
+    if not user_in_docker_group:
+        return_code = subprocess.call(
+            ["docker", "ps"],
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+            timeout=30
+        )
+        print("docker ps returned: {}".format(return_code))
+        if return_code != 0:
+            use_sudo = True
```

Comment:
checking `docker ps` feels like it should be in it's own function too

Reply:
Done